rs/sns/governance/canister/governance.did

```diff
@@ -12,9 +12,11 @@ type Action = variant {
   UpgradeSnsToNextVersion : record {};
   AdvanceSnsTargetVersion : AdvanceSnsTargetVersion;
   RegisterDappCanisters : RegisterDappCanisters;
+  RegisterTreasuryManager : RegisterTreasuryManager;
   TransferSnsTreasuryFunds : TransferSnsTreasuryFunds;
   UpgradeSnsControlledCanister : UpgradeSnsControlledCanister;
   DeregisterDappCanisters : DeregisterDappCanisters;
+  TreasuryManagerWithdraw : TreasuryManagerWithdraw;
```

Comment:
This keeps the canister running under SNS control for supporting audits after withdrawal. 

The treasury manager canister can then be de-registered via a subsequent DeregisterDappCanisters.